docs/datadog_timber_module.md

```diff
@@ -0,0 +1,30 @@
+# Datadog Timber Integration Package
```

Comment:
```# Android Timber Integration```

Also, instead of `datadog_timber_module.md` rename this file `timber_integration.md`